### ISSUE TITLE
Investigate focus issues on the claim status list

### ISSUE THREAD

**Issue by seesleestak** [department-of-veterans-affairs/va.gov-team#129075]
## Context

ADE specialists have raised focus issues for the claim status list in staging. [DSVA Slack thread here](https://dsva.slack.com/archives/C05UTPZRZFY/p1767718284335449).

Testing we determined that this seems to be an issue on older versions of Safari. Investigate if this is indeed the case and if there is anything that we can do about it.

It sounds like this issue is also happening on the MHV homepage so maybe this issue is something that is occurring on multiple places due to the version of Safari that someone has.

## Description

When tabbing through the page focus moves from the header to filter dropdown and then past all the claim cards to the footer. All the links into the claim cards need to be keyboard accessible.

## Acceptance criteria
- [ ] Investigate if there is a way to fix this on older Safari versions
